src/kubernetes-tool/templates/splash_screen.vue

```diff
@@ -33,15 +33,30 @@ limitations under the License.
                 <div class="input-container">
                     <label for="helmInput" class="hidden">{{ i18n.templates.splashScreen.helmTitle }}</label>
                     <i class="fas fa-dharmachakra"></i>
-                    <input id="helmInput"
-                           v-model="helmId"
-                           class="input"
-                           type="text"
-                           :placeholder="i18n.templates.splashScreen.helmTitle"
-                    />
-                    <button id="submitHelm" class="button is-primary" :click="submitHelm">
-                        {{ i18n.templates.splashScreen.submit }}
-                    </button>
+                    <vue-autosuggest
+                        v-model="helmId"
+                        :suggestions="helmSuggestions"
+                        :input-props="{id: 'helmInput', class: 'input', type: 'text', placeholder: i18n.templates.splashScreen.helmTitle, style: {width: '80%', textIndent: '30px'}, readonly: readonly}"
```

Comment:
Not a fan of how long this line is, can we wrap it here?
If not, can we move it to data so that it can be wrapped nicely?